roles/gcs_backend/tasks/delete.yml

```
@@ -0,0 +1,19 @@
+---
+- name: Delete a storage bucket
+  google.cloud.gcp_storage_bucket:
+    name: "{{ gcs_backend_bucket_name }}"
+    project: "{{ gcs_backend_gcp_project }}"
+    auth_kind: "{{ gcs_backend_auth_kind }}"
+    service_account_file: "{{ gcs_backend_service_account_file }}"
+    state: absent
+  ignore_errors: true  # currently module `gcp_storage_bucket` does not support deleting non-empty buckets
+  register: delete_bucket_result
+
+- name: Display message in case of bucket deletion failure
+  ansible.builtin.fail:
+    msg:
+      - "Deleting non-empty buckets might fail currently and buckets should be deleted manually."
+      - "Please refer to the following error message for more details."
+      - "{{ delete_bucket_result.msg }}"
+  when: delete_bucket_result is failed
+  ignore_errors: true
```

Hmm, I don't think we want to ignore errors here. This is in fact a failure, and should be presented to the user as such.